runtime/v2/module.go

```diff
@@ -241,22 +222,6 @@ func ProvideEnvironment[T transaction.Tx](
 		KVStoreService:     kvService,
 		MemStoreService:    memKvService,
 	}
-
-	return env, kvService, memKvService
-}
-
-func registerStoreKey[T transaction.Tx](builder *AppBuilder[T], key string) {
-	builder.app.storeKeys = append(builder.app.storeKeys, key)
-}
-
-func storeKeyOverride(config *runtimev2.Module, moduleName string) *runtimev2.StoreKeyConfig {
```

Comment:
Ditto, where is this functionality? Would auth still have its store key set as `acc` as defined in the app config?

Reply:
function is still there.. the diff has it moved for some reason

Reply:
my bad, you don't see that easily on a phone 👍🏾